packages/arb-token-bridge-ui/tests/e2e/specs/withdraw.cy.ts

```diff
@@ -0,0 +1,140 @@
+
+      cy.findByRole('button', {
+        name: /Move funds to Goerli/i
+      })
+        .should('be.visible')
+        .should('be.disabled')
+    })
+
+    context("bridge amount is lower than user's L2 ETH balance value", () => {
+      const zeroToLessThanOneETH = /0(\.\d+)*( ETH)/
+      it('should show summary', () => {
+        cy.findByPlaceholderText('Enter amount')
+          .type('0.0001', { scrollBehavior: false })
+          .then(() => {
+            cy.findByText('You’re moving')
+              .siblings()
+              .last()
+              .contains('0.0001 ETH')
+              .should('be.visible')
+            cy.findByText('You’ll pay in gas fees')
+              .siblings()
+              .last()
+              .contains(zeroToLessThanOneETH)
+              .should('be.visible')
+            cy.findByText('L1 gas')
+              .parent()
+              .siblings()
+              .last()
+              .contains(zeroToLessThanOneETH)
+              .should('be.visible')
+            cy.findByText('L2 gas')
+              .parent()
+              .siblings()
+              .last()
+              .contains(zeroToLessThanOneETH)
+              .should('be.visible')
+            cy.findByText('Total amount')
+              .siblings()
+              .last()
+              .contains(/(\d*)(\.\d+)*( ETH)/)
+              .should('be.visible')
+          })
+      })
+
+      it('should show a clickable withdraw button', () => {
+        cy.findByRole('button', {
+          name: /Move funds to Goerli/i
+        })
+          .should('be.visible')
+          .should('not.be.disabled')
+          .click({ scrollBehavior: false })
+      })
+
+      it('should show withdrawal confirmation', () => {
+        cy.findByText(/Use Arbitrum’s bridge/i).should('be.visible')
+
+        // the Continue withdrawal button should be disabled at first
+        cy.findByRole('button', {
+          name: /Continue/i
+        }).should('be.disabled')
+
+        cy.findByRole('switch', {
+          name: /before I can claim my funds/i
+        })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+
+        cy.findByRole('switch', {
+          name: /after claiming my funds/i
+        })
+          .should('be.visible')
+          .click({ scrollBehavior: false })
+          .then(() => {
+            // the Continue withdrawal button should not be disabled now
+            cy.findByRole('button', {
+              name: /Continue/i
+            })
+              .should('not.be.disabled')
+              .click({ scrollBehavior: false })
+          })
+      })
+
+      it('should withdraw successfully', () => {
+        cy.confirmMetamaskTransaction().then(() => {
+          cy.findAllByText(/Moving 0.0001 ETH to Goerli/i).should('be.visible')
+        })
+      })
+    })
+
+    // TODO => test for bridge amount higher than user's L2 ETH balance
```

Comment:
we can add one more `TODO` for the case when user's wallet doesn't have L2 on the network lists so clicking the `Move funds to Goerli` button will prompt the user to add and switch to L2 network to proceed (the test scenario you originally tested for)

I would say that test scenario is also a happy path so it could be included in this PR? Or maybe we can do a withdrawal happy path 2 PR?

Reply:
that's a good idea - @dewanshparashar let's add a todo comment for now and we'll tackle it later